doc/dictionary-travis-ci.md

```diff
@@ -0,0 +1,72 @@
+# TL;DR
+Configure continuous integration (test, build, deploy) of a Gen3 dictionary with Travis CI.
+## Prerequisites
+* Public dictionary repository (e.g. [bioteam/dictionaryutils](https://github.com/bioteam/dictionaryutils.git))
```

Comment:
this repo doesn't seem to be public, let's replace the link with one of our dictionaries, maybe https://github.com/uc-cdis/canine_dictionary

Reply:
I can see bioteam repo without logging in. I agree that canine dictionary is also a good example.

Reply:
yeah i can also see it now. i would still rather link to a dictionary we own, and for now the demo dictionary is canine